int/int_test.go

```diff
@@ -112,7 +111,7 @@ var _ = Describe("Integrationtests", func() {
 			err := i.RemoveRouteMonitor(routeMonitorNamespace, routeMonitorName)
 			routeMonitorName = "fake-route-monitor"
 			routeMonitorNamespace = "default"
-			expectedServiceMonitorName = templates.TemplateForServiceMonitorName(routeMonitorNamespace, routeMonitorName)
+			expectedServiceMonitorName = types.NamespacedName{Name: routeMonitorName, Namespace: routeMonitorNamespace}
```

Comment:
and you would just need to change from where we pull the resource
IIRC
```suggestion
			expectedServiceMonitorName = v1alpha1.NamespacedName{Name: routeMonitorName, Namespace: routeMonitorNamespace}
```

Reply:
nvm, resending my comment